tests/e2e/config/environment.js

```diff
@@ -6,9 +6,17 @@ const { addAttach } = require( 'jest-html-reporters/helper' );
 
 class E2EEnvironment extends PuppeteerEnvironment {
 	async handleTestEvent( event ) {
-		if ( event.name === 'test_fn_failure' ) {
-			const data = await this.global.page.screenshot();
-			await addAttach( data, 'Full Page Screenshot', this.global );
+		if (
+			event.name === 'test_fn_failure' ||
+			event.name === 'hook_failure'
+		) {
+			const attach = await this.global.page.screenshot();
```

Comment:
I think a full-page screenshot here will me more useful:
```suggestion
			const attach = await this.global.page.screenshot( {
				fullPage: true,
			} );
```

Reply:
Using `fullPage: true` stops it from capturing the right page for me, it should be capturing the back-end WC settings page for the case I tested with, but `fullPage: true` makes it capture the front-end cart page?

Also, I think in the artifact with missing images, this might be caused by the error happening in the `beforeAll` hook, it seems like the error is happening before it even gets into a test case which is why the image doesn't attach.

I made it so it only takes a full page screenshot when the event name is not `hook_failure`. This seems to show us the right page in `beforeAll`, `before`, `after`, and `afterAll` errors, and the right page in regular test failures.

Reply:
Also, what are you changing locally to get this:
>Testing locally with a failed test case, I see the image in the jest-html-reporters-attach folder but I don't see it in the HTML report.

Thanks!